Enable compile without tcp-wrappers

Patch by Timothy Redaelli <drizzt@gentoo.org>

http://bugs.gentoo.org/178242

--- portmap_6.0/pmap_check.c
+++ portmap_6.0/pmap_check.c
@@ -44,7 +44,9 @@
 #include <netinet/in.h>
 #include <rpc/rpcent.h>
 #endif
+#ifdef HOSTS_ACCESS
 #include <tcpd.h>
+#endif
 #include <arpa/inet.h>
 #include <grp.h>
 
